dags/dim_active_users_aggregates.py

```diff
@@ -61,7 +61,7 @@
     with DAG(
         dag_id,
         default_args=default_args,
-        schedule_interval="30 4 * * *",
+        schedule_interval="15 4 * * *",
```

Comment:
The aggregates are now running at 4:15 so the same schedule wouldn't work, that's what is causing the conflict now. Should this run after or before?

Reply:
Ref. https://github.com/mozilla/bigquery-etl/blob/65f7f55848e09abf556a68429f3d298cbd961734/dags.yaml#L816

Reply:
Ehmm, why would it not work? the execution delta for checking for external task completion has been updated to be `0` so the schedules between the two DAGs needs to match as far as I understand:
https://github.com/mozilla/telemetry-airflow/pull/1846/files#diff-8b74dd31a49290bfa268f73dd62b36b88a3d1e09cae8e232ebcdfa377461ef84R78

Reply:
Match meaning that both DAGS run at the same time, correct?
In that case wouldn't the checks fail if they run before the aggregates are fully updated, therefore blocking the KPI dashboard?

Reply:
Is 4:30 a better time to run the checks if the kpi_dashboard is moved to 4:45?

Reply:
The checks will not run before the upstream task as the sensor checks for the task successful completion and only then the checks are executed.